internal/cli/atlas/metrics/databases/describe.go

```diff
@@ -59,14 +59,22 @@ func (opts *DatabasesDescribeOpts) Run() error {
 	return opts.Print(r)
 }
 
-// mcli om metric(s) disk(s) describe <hostId:port> <name> --granularity g --period p --start start --end end [--type type] [--projectId projectId].
-func DatabasesDescribeBuilder() *cobra.Command {
+// mcli atlas metric(s) database(s) describe <host:port> <databaseName> --granularity g --period p --start start --end end [--type type] [--projectId projectId].
+func DescribeBuilder() *cobra.Command {
 	const argsN = 2
-	opts := &DatabasesDescribeOpts{}
+	opts := &DescribeOpts{}
 	cmd := &cobra.Command{
-		Use:   "describe <hostId> <name>",
+		Use:   "describe <host:port> <databaseName>",
```

Comment:
Probably a good idea to use the `Long` property to add an explanation on how to get the hostname and port

Reply:
I don't see the Long property. Should we add it?

Reply:
Sure, Could you tell me what to add? the commands to retrieve hostname and port is `mongocli atlas process list`